mlflow/transformers/inference_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+_INFERENCE_TASK_COMPLETIONS = "llm/v1/completions"
+_INFERENCE_TASK_CHAT = "llm/v1/chat"
+
+
+def _get_output_and_usage_from_tensor(
+    prompt: str, output_tensor: List[int], pipeline, model_config, inference_task
+):
+    """
+    Decode the output tensor and return the output text and usage information as a dictionary.
```

Comment:
nit: could you add a mention that this is to make output OpenAI compatible format.